packages/client/hmi-client/src/components/navbar/tera-notification-panel.vue

```diff
@@ -103,6 +98,9 @@ const getTitleText = (item: NotificationItem) => {
 	}
 };
 
+const isStatusCorrect = (status: ItemStatusType, item: NotificationItem) =>
+	status === ItemStatusType[item.status];
+
```

Comment:
What I meant is simpler function for readability, this is not easier to read
```suggestion
const isComplete = (item: NotificationItem) => item.status === ProgressState.Complete;
const isFailed = (item: NotificationItem) => item.status === ProgressState.Failed;
const isRunning = (item: NotificationItem) => item.status === ProgressState.Running;
```

Reply:
This still needs to be resolved